### PR TITLE
[feature](udf) Support for IP types in Java UDF.

### PR DESCRIPTION
**Pull request by Mryange**
### What problem does this PR solve?

Add support for IPV types in Java UDFs.
doc https://github.com/apache/doris-website/pull/1444

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [x] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run buildall

**Comment by Mryange** [comment]
run p0

**Comment by Mryange** [comment]
run buildall

**Review by zhangstar333 — DISMISSED** [review]

File: regression-test/java-udf-src/src/main/java/org/apache/doris/udf/IPV4TypeTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            return "null";
+        }
+        return x.toString();
+    }
+
+    // output ipv4
+    public InetAddress evaluate(String s) {
+        try {
+            InetAddress ipv4Address = InetAddress.getByName(s);
+            if(ipv4Address.getAddress().length == 4){
```
Comment (HappenLee) [inline regression-test/java-udf-src/src/main/java/org/apache/doris/udf/IPV4TypeTest.java:36]: code format

**Comment by Mryange** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Comment by Mryange** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**